shacl12-common/local-biblio.js

```diff
@@ -0,0 +1,109 @@
+var localBibliography = {
+  ISO24707: {
+    id: "ISO24707",
+    title: "Information technology — Common Logic (CL) — A framework for a family of logic-based languages",
+    date: "2018-07",
+    href: "https://www.iso.org/standard/66249.html",
+    status: "Published",
+    publisher: "ISO/EIC"
+  },
+  HORST04: {
+    id: "HORST04",
+    title: "Extending the RDFS Entailment Lemma",
+    authors: ["Herman J. ter Horst"],
+    date: "2004",
+    isbn: "978-3-540-30475-3",
+    href: "https://doi.org/10.1007/978-3-540-30475-3_7",
+    status: "Conference Proceeding",
+    publisher: "Springer, Berlin, Heidelberg"
+  },
+  HORST05: {
+    id: "HORST05",
+    title: "Completeness, Decidability and Complexity of Entailment for RDF Schema and a Semantic Extension Involving the OWL Vocabulary",
+    authors: ["Herman J. ter Horst"],
+    date: "2005",
+    href: "https://dx.doi.org/10.2139/ssrn.3199251",
+    status: "Conference Proceeding",
+    publisher: "Journal of Web Semantics"
+  },
+  SAFETY: {
+    title: "Saftety of recursive horn clauses with infinite relations",
+    href: "http://arxiv.org/abs/cs.LO/9809032",
+    authors: [
+      "R. Ramakrishnan",
+      "F. Bancilhon",
+      "A. Silberschatz"
+    ],
+    date: "1987",
+    publisher: "ACM New York"
+  },
```

Comment:
The arxiv link and 1987 date sent me looking for an upstream citation.

```suggestion
  SAFETY: {
    title: "Safety of recursive horn clauses with infinite relations",
    href: "https://doi.org/10.1145/28659.28694",
    authors: [
      "R. Ramakrishnan",
      "F. Bancilhon",
      "A. Silberschatz"
    ],
    date: "1987",
    publisher: "ACM New York"
  },
```